charts/stable/vaultwarden/values.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  repository: vaultwarden/server
+  pullPolicy: IfNotPresent
+  tag: latest
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the Vaultwarden documentation
+# https://github.com/dani-garcia/vaultwarden/blob/main/.env.template
+
+mariadb:
+  enabled: false
+
+postgresql:
+  enabled: false
```

Comment:
These are almost always put at the bottom of the Values.yaml file.

Also:
Would be nice to add a comment refering to the BitNami Docs and/or some (commented out) defaults